### PR TITLE
Update deploy md doc adding the correct publish directory folder on Netlify settings

### PR DESCRIPTION
**Pull request by WagnerMoreira**
The purpose of this pr is to update the deploy docs adding the correct `publish directory` folder when doing the Netlify Continuous deploy setup. The correct directory is  `.storybook-static` and not `storybook-static` as it says in the current version and the image.
I had this issue today when I was doing this setup for the first time, I could not find the solution anywhere and I solved by debugging it myself, so I thought it could be valuable to update the docs.

**Pending questions:** 

1. The docs are pointing to image files that are not in this repo, therefore I can't edit them to add the `.`. how can I proceed with that?

2. I also had to use the `--quiet` flag on the `build-storybook` command, otherwise, the builds on Netlify were failing because it would get stuck around 70% and it was because it was generating too many log outputs. Can I also add this information to this doc?

### CODE REVIEW

**Comment by jonniebigodes** [comment]
Good catch. The current images and text are a bit outdated so it seems. Updating them should be a good improvement to the documentation. 
Regarding your questions:

1- You can fork the repository and update the images, and create a pull request addressing this,  or if you don't mind waiting a bit me or @domyen or @tmeasday or @kylesuss can make the image update.

2- To avoid issues like the one you've experienced with netlify builds failing a good improvement to that section of the tutorial could be the following, right under the image that needs to be replaced:

```html
<div class="aside"><p>Should your deployment fail with netlify, add the following flag to your <code>build-storybook script</code><a href="https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook">--quiet </a></p></div>
```
Sounds good?

**Comment by WagnerMoreira** [comment]
@jonniebigodes Thanks! 

1. I don't see any images in this repo, are they located in another repo or you mean I can get them from the learnstorybook.com by inspecting and them add the files to this repo? I can definitely do that, just need to confirm this info before doing that.

2. Sounds good, I will do update this PR adding that soon 👍

**Comment by WagnerMoreira** [comment]
@jonniebigodes also, could you assign me to this PR? I don't see the option to assign myself.

**Comment by domyen** [comment]
Thanks for the initiative @WagnerMoreira, the images should be in /static/intro-to-storybook

**Comment by WagnerMoreira** [comment]
@domyen thanks for assigning me and for the file path!

**Comment by WagnerMoreira** [comment]
@jonniebigodes @domyen just updated the PR with a few more additions, I've updated en and pt lang files in this last commit https://github.com/chromaui/learnstorybook.com/pull/222/commits/7f535c1742f95beb5ce9dc656177bc40e2ccdcfb pt is my main lang and en second 😄

File: content/intro-to-storybook/angular/en/deploy.md [hunk 1]
```diff
@@ -84,10 +84,12 @@ Now select the taskbox GitHub repo from the list of options.
 
 ![Netlify connect to repo](/intro-to-storybook/netlify-account-picker.png)
 
-Configure Netlify by highlighting which build command to run in its CI and which directory the static site is outputted in. For branch choose `master`. Directory is `storybook-static`. Build command use `yarn build-storybook`.
+Configure Netlify by highlighting which build command to run in its CI and which directory the static site is outputted in. For branch choose `master`. Directory is `.storybook-static`. Build command use `yarn build-storybook`.
 
 ![Netlify settings](/intro-to-storybook/netlify-settings.png)
 
+<div class="aside"><p>Should your deployment fail with Netlify, add the <a href="https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook">--quiet </a> flag to your <code>build-storybook</code> script.</p></div>
```
Comment (WagnerMoreira) [inline content/intro-to-storybook/angular/en/deploy.md]: @jonniebigodes added the text following your suggestion 👍

File: content/intro-to-storybook/angular/pt/deploy.md [hunk 2]
```diff
@@ -85,10 +85,12 @@ Seguida da seleção do repositório da lista de opções.
 
 ![Conexão Netlify para o repositorio](/intro-to-storybook/netlify-account-picker.png)
 
-É feita a configuração no Netlify ao selecionar-se o comando apropriado para executar no IC(CI na forma nativa) e qual a pasta de output. Como ramo, seleciona-se `master`. Pasta `storybook-static`. Comando `yarn build-storybook`.
+É feita a configuração no Netlify ao selecionar-se o comando apropriado para executar no IC(CI na forma nativa) e qual a pasta de output. Como ramo, seleciona-se `master`. Pasta `.storybook-static`. Comando `yarn build-storybook`.
 
 ![Configurações Netlify](/intro-to-storybook/netlify-settings.png)
 
+<div class="aside"><p>Caso o deploy no Netlify falhe, adicione a <a href="https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook">--quiet </a> flag ao seu comando <code>build-storybook</code>.</p></div>
```
Comment (jonniebigodes) [inline content/intro-to-storybook/angular/pt/deploy.md]: @WagnerMoreira remove the `seu` from here, leave it only as `adicione a flag ao comando.....`. To make it neutral and more approachable to both portuguese speaking audiences.

Comment (WagnerMoreira) [reply content/intro-to-storybook/angular/pt/deploy.md]: @jonniebigodes  thanks! updated

Comment (jonniebigodes) [reply content/intro-to-storybook/angular/pt/deploy.md]: No problem.

**Comment by WagnerMoreira** [comment]
@jonniebigodes @domyen  Hi 👋 is this PR good to be merged? Please let me know if you need anything else.

**Comment by jonniebigodes** [comment]
Been through it and seems good to go.

**Comment by domyen** [comment]
Nice thank you

**Pull request merged**